ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -71,7 +77,11 @@
       <p>{{dc}}</p>
     {{#each upstreams as |item|}}
       <TopologyMetrics::Card
+<<<<<<< HEAD
         @item={{item}}
+=======
+        @items={{upstreams}}
+>>>>>>> 56c2ff56e (ui: Update to not return metrics for ingress gateways (#9081))
```

Comment:
`as |item|` in the loop appears to be completely unused if taking this change, not sure what's intended